### DISCUSSION TITLE
The dpgen potential function training problem

### DISCUSSION THREAD

**Discussion by Edinbur** [deepmodeling/deepmd-kit#3591] [Q&A]
Wanted to ask a dpgen question
When I finished the first round of iteration, the trained energy error is very small, the force error is average, but after model deviation to add new data to train again, the trained potential function energy and force error is two orders of magnitude larger than the first time, resulting in the lammps directly running crash when I do the second round of model deviation, can anyone help me to answer and solve the problem please?

**Comment by njzjz** [comment]
Could you check whether old and new data use the same level of theory?

**Reply by Edinbur** [reply]
Yes, I just changed INCAR to a single point energy calculation, besides that, the initial AIMD training set is 192 atoms and a single layer structure, which I used during the training process, and I don't know if it is possible to train with this 192 atom system? Or can I only use a single cell?
